make/gen.cmake

```diff
@@ -29,8 +29,22 @@ function(V2X)
   )
 
   set(INCLUDES "")
-
   set(DEPENDS_LIST "")
+  set(VERILOG_DEPS "")
+
+  if(V2X_GENERATEDXMLDEPS)
+    list(APPEND INCLUDES ${V2X_GENERATEDXMLDEPS})
```

Comment:
Are these absolute file locations or file targets?

Reply:
file targets

Reply:
Rather than pass the file targets as arguments, why not store the relevant absolute file locations in the SRCS target priority INCLUDE_FILES, and then append_file_dependencies will correctly generate the transitive dependency list.  The output XML will need to have their INCLUDE_FILES property set, but that is required not match whether that is done implicitly or explicitly.